backend/main/src/Group/UseCases/CreateContainerUseCase/ICreateContainerUseCase.ts

```diff
@@ -0,0 +1,8 @@
+import { UseCaseError } from "src/Shared";
+import { ContainerDomainError } from "src/Group/Domain/Entities/Container";
+
+export interface ICreateContainerUseCase {
```

Comment:
The container must be bound to a group. Therefore, this use case needs to get a Group ID.